searchlib/src/vespa/searchlib/attribute/multinumericenumattribute.hpp

```diff
@@ -145,5 +146,29 @@ MultiValueNumericEnumAttribute<B, M>::getSearch(QueryTermSimple::UP qTerm,
     return std::make_unique<attribute::MultiNumericEnumSearchContext<T, M>>(std::move(qTerm), *this, this->_mvMapping.make_read_view(doc_id_limit), this->_enumStore);
 }
 
+template <typename B, typename M>
+long
+MultiValueNumericEnumAttribute<B, M>::onSerializeForAscendingSort(DocId doc, void* serTo, long available, const common::BlobConverter*) const
+{
+    attribute::NumericSortBlobWriter<T, true> writer;
```

Comment:
Consider adding a helper function so lines 153-158 can be shared between this function and _onSerializeForDescendingSort()_.

Reply:
Done.